js/tc39/breaking_test_errors.json

```diff
@@ -2145,6 +2139,7 @@
   "test/language/expressions/import.meta/distinct-for-each-module.js-strict:true": "test/language/expressions/import.meta/distinct-for-each-module.js: SyntaxError: test/language/expressions/import.meta/distinct-for-each-module.js: Unexpected token (29:20)\n  27 | \n  28 | // The imported module has a distinct import.meta object.\n\u003e 29 | assert.notSameValue(import.meta, fixture_meta,\n     |                     ^\n  30 |                     \"foreign import.meta accessed via import binding\");\n  31 | assert.notSameValue(import.meta, getMeta(),\n  32 |                     \"foreign import.meta accessed via function call\"); at \u003ceval\u003e:2:28542(114)",
   "test/language/expressions/import.meta/import-meta-is-an-ordinary-object.js-strict:true": "test/language/expressions/import.meta/import-meta-is-an-ordinary-object.js: SyntaxError: test/language/expressions/import.meta/import-meta-is-an-ordinary-object.js: Unexpected token (28:24)\n  26 | \n  27 | // import.meta is an object.\n\u003e 28 | assert.sameValue(typeof import.meta, \"object\",\n     |                         ^\n  29 |                  \"typeof import.meta is 'object'\");\n  30 | assert.notSameValue(import.meta, null,\n  31 |                     \"typeof import.meta is 'object' and import.meta isn't |null|.\"); at \u003ceval\u003e:2:28542(114)",
   "test/language/expressions/import.meta/same-object-returned.js-strict:true": "test/language/expressions/import.meta/same-object-returned.js: SyntaxError: test/language/expressions/import.meta/same-object-returned.js: Unexpected token (28:8)\n  26 | ---*/\n  27 | \n\u003e 28 | var a = import.meta;\n     |         ^\n  29 | var b = function() { return import.meta; }();\n  30 | \n  31 | assert.sameValue(import.meta, a, at \u003ceval\u003e:2:28542(114)",
+  "test/language/expressions/import.meta/syntax/escape-sequence-import.js-strict:true": "test/language/expressions/import.meta/syntax/escape-sequence-import.js: error is not an object (Test262: This statement should not be evaluated.)",
```

Comment:
testing whether `import` in `import.meta` can be escaped and work - it shouldn't , but it does - I don't see that as a ... problem :)